index.html

```diff
@@ -289,7 +289,114 @@ <h2>About me</h2>
           </ul>
         </div>
       </div>
+       <div id="contact">
+        <h2>
+          I'm always interested in hearing about new projects, so if you'd like
+          to chat please get in touch.
+        </h2>
+        <form  id ="form" action="https://formspree.io/f/mwkakrqj" method="GET">
```

Comment:
Make sure the form method is `POST` because `Formspree` is listening to a `POST` method not a `GET` method.